### PR TITLE
Updated cost tags specifications

### LINKED ISSUE #9: Investigate why tags are not being set via MC
Cost tags are not being set when the Unity UI is installed via the Management Console, but they are working properly when the application is installed when running the terraform code locally.

### PR DESCRIPTION
**Pull request by anilnatha**
## Purpose

This PR addresses the need to revert costs tags to the original setup after extensive testing with MC.

## Proposed Changes
- [CHANGE] Cost tags to use original specification that was implemented after it was determined that the newest version of MC doesn't experience the issues we were encountering with cost tags not being set.

## Issues

Fixes #9 

## Testing

Tested with Galen using a custom project venue he had set up.

### CODE REVIEW

**Comment by anilnatha** [comment]
Note: I ran this terraform locally and observed tags were set properly, so syntactically, the configuration is correct.

**Review by rtapella — APPROVED** [review]
looks good

File: terraform-unity-ui/variables.tf [hunk 1]
```diff
@@ -34,6 +34,21 @@ variable "tags" {
   type        = map(string)
 }
 
+variable "additional_tags" {
+  description = "Additional Resource Tags"
+  type        = map(string)
+  default     = {
+    Venue = "dev"
```
Comment (rtapella) [inline terraform-unity-ui/variables.tf:41]: inject with MGMT console

File: terraform-unity-ui/variables.tf [hunk 2]
```diff
@@ -34,6 +34,21 @@ variable "tags" {
...
+    ServiceArea = "uiux"
+    CapVersion = "0.8.0"
+    Component = "navbar"
+    Proj = "unity"
+    CreatedBy = "uiux"
+    Env = "dev"
```
Comment (rtapella) [inline terraform-unity-ui/variables.tf:47]: inject with mgmt consoel

**Review by rtapella — APPROVED** [review]

**Review by galenatjpl — COMMENTED** [review]
looks good to me!

**Pull request merged**